src/index.ts

```diff
@@ -0,0 +1,44 @@
+import yargs from 'yargs';
+import { ESOut, Stdout, docBuffer } from './dumper';
+import { loadConfig } from './config';
+import {SpinnerNotifier} from "./notifier";
+
+export default async function main() {
+    const argv = yargs
+        .usage('$0 <command> [options]')
+        .options({
+            config: {
+                alias: 'c',
+                describe: 'config path',
+                default: './config.yaml',
```

Comment:
I suggest this default file path be made absolute to the config under this library and not the path from where the command is being run (`cwd`).  Also, use `path.join()` to make it compatible with multiple OSs (aka: Windows :) ).  This will (I think) enable us to then be able to run this tool via `npx` 😃 

Change to:

```javascript
default: path.join(__dirname, "config.yaml")
```

Reply:
Do we have people with windows. I can make this change if needed.

Reply:
The comment applies regardless of windows :)

but yes - I'm on Windows